_posts/2021-06-28-regular-expressions-in-ruby.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: post
+title:  "Regular expressions in Ruby"
+date:   2021-06-28 19:00:00 +0200
+categories: []
```

Comment:
I think we could start adding categories... So the posts will be easier to categorize

```suggestion
categories: [ruby, regex]
```